docs/content/en/docs/implementing/otel.md

```diff
@@ -4,26 +4,193 @@ description: How to standardize access to OpenTelemetry observability data
+[Standardize observability](../getting-started/observability)
+getting started guide.
+
+## DORA metrics
+
+DORA metrics are an industry-standard set of measurements;
+see the following for a description:
+
+- [What are DORA Metrics and Why Do They Matter?](https://codeclimate.com/blog/dora-metrics)
+- [Are you an Elite DevOps Performer?
+   Find out with the Four Keys Project](https://cloud.google.com/blog/products/devops-sre/using-the-four-keys-to-measure-your-devops-performance)
+
+DORA metrics provide information such as:
+
+- How many deployments happened in the last six hours?
+- Time between deployments
+- Deployment time between versions
+- Average time between versions.
+
+The Keptn Lifecycle Toolkit starts collecting these metrics
+as soon as you apply
+[basic annotations](integrate/#basic-annotations)
+to the `Deployment` resource.
+Metrics are collected only for the `Deployment` resources
+that are annotated.
+
+To view DORA metrics, run the following command:
+
+```shell
+kubectl port-forward -n keptn-lifecycle-toolkit-system \
+   svc/lifecycle-operator-metrics-service 2222
+```
+
+Then view the metrics at:
+
+```shell
+http://localhost:2222/metrics
+```
+
+DORA metrics are also displayed on Grafana
+or whatever dashboard application you choose.
+
+## OpenTelemetry
+
+### Requirements for OpenTelemetry
+
 To access OpenTelemetry metrics with the Keptn Lifecycle Toolkit,
-you must:
+you must have the following on your cluster:
 
-- Install an OpenTelemetry collector on your cluster.
+- An OpenTelemetry collector.
   See
   [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/)
   for more information.
+- Prometheus Operator.
+  See [Prometheus Operator Setup](https://github.com/prometheus-operator/kube-prometheus/blob/main/docs/customizing.md).
+- The Prometheus Operator must have the required permissions
+  to watch resources of the `keptn-lifecycle-toolkit-system` namespace (see
+  [Setup for Monitoring other Namespaces](https://prometheus-operator.dev/docs/kube/monitoring-other-namespaces/).
+
+If you want a dashboard for reviewing metrics and traces,
+you need:
+
+- [Grafana](https://grafana.com/)
+  or the dashboard of your choice.
+   See
+  [Grafana Setup](https://grafana.com/docs/grafana/latest/setup-grafana/).
+
+- [Jaeger](https://jaegertracing.io)
+  or a similar tool if you want traces.
+   See
+  [Jaeger Setup](https://github.com/jaegertracing/jaeger-operator).
+
+If you do not already have an installation of the Jaeger
+[manifest](https://github.com/jaegertracing/jaeger-operator/releases/download/v1.38.0/jaeger-operator.yaml),
+use the following commands to
+install Jaeger into the `observability` namespace
+and the Jaeger resource into the `lifecycle-toolkit` namespace:
+
+```shell
+kubectl create namespace observability
+kubectl apply -f \
+    https://github.com/jaegertracing/jaeger-operator/releases/download/v1.38.0/jaeger-operator.yaml \
+    -n observability
```

Comment:
I agree.  I almost deleted it last night but, since Thomas had it in the README, I pulled it over to  see how it would fare in review ;-)  Happy to delete -- we don't need to be maintaining docs for other project's software, right?